tpot/tpot.py

```diff
@@ -113,7 +113,8 @@ def __init__(self, population_size=100, generations=100,
         self.pset.addPrimitive(self.gradient_boosting, [pd.DataFrame, float, int, int], pd.DataFrame)
         self.pset.addPrimitive(self._combine_dfs, [pd.DataFrame, pd.DataFrame], pd.DataFrame)
         self.pset.addPrimitive(self._variance_threshold, [pd.DataFrame, float], pd.DataFrame)
-        self.pset.addPrimitive(self._select_kbest, [pd.DataFrame, int], pd.DataFrame) 
+        self.pset.addPrimitive(self._select_kbest, [pd.DataFrame, int], pd.DataFrame)
+        self.pset.addPrimitive(self._select_fwe, [pd.DataFrame, int], pd.DataFrame)
```

Comment:
Just noticed -- this should be

```python
 self.pset.addPrimitive(self._select_fwe, [pd.DataFrame, float], pd.DataFrame)
```

since the second parameter that `select_fwe()` takes is the alpha value in the range [0.001, 0.05].